docs/src/styles/docs/toc.scss

```diff
@@ -21,10 +21,12 @@
 .docs-toc-inner {
   position: sticky;
   width: 16rem;
+  height: 80vh;
   flex-shrink: 0;
   top: 8rem;
   right: 0;
   padding: var(--amplify-space-medium);
+  overflow-y: auto;
```

Comment:
We have a [`ScrollView`](https://ui.docs.amplify.aws/components/scrollview) primitive out of the box. We can probably change the layout container to it and save this line 😄